setup.py

```diff
@@ -39,6 +39,7 @@ def get_install_requires() -> List[str]:
         "scipy!=1.4.0",
         "sqlalchemy>=1.1.0",
         "tqdm",
+        "dm-haiku @ git+https://github.com/deepmind/dm-haiku",
```

Comment:
```suggestion
```

I think any library required only by example shouldn't be listed here.

Reply:
Thanks!